staging/src/k8s.io/apiserver/pkg/registry/generic/rest/proxy.go

```diff
@@ -78,7 +78,7 @@ func NewUpgradeAwareProxyHandler(location *url.URL, transport http.RoundTripper,
 
 // ServeHTTP handles the proxy request
 func (h *UpgradeAwareProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	if len(h.Location.Scheme) == 0 {
+	if h.Location.Scheme == "" {
```

Comment:
While I prefer it the way you changed it, I think the style guide (admittedly, mostly a theoretical document) has it the old way.